_posts/2018-01-02-getting-up-and-running-with-react-native.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: post
+title: Getting up and running with React Native
+date: 2018-01-02
+---
+
+Today, I meant to look into [react-native](https://facebook.github.io/react-native/). I know React (a little) but recently realised I had made some wrong assumptions about React Native: I thought the idea behind React and React-native was: **Learn once, write once, use everywhere**. It is NOT! The real motto is: **Learn once, use everywhere**, i.e. you learn the basics once but you can't have one code base targeting all platforms. What a disappointment... I thought with the help of some kind of web dark magic it was the case.
+
+Anyway, since it is not the case and one needs to learn a few things that are React Native specific, off I went to learn the basic of React Native. It actually took me quite a while to get up and running as I met quite a few hick ups along the way.
```

Comment:
hiccups*

Reply:
oops, I picked the wrong correction with my spell checker ^^